src/plugins/event-plugins/DomEventPlugin.ts

```diff
@@ -113,15 +119,33 @@ export class DomEventPlugin implements Plugin {
         return '';
     }
 
+    private getElementCSSLocator(event: Event, domEvent: TargetDomEvent) {
+        if (!event.target) {
+            return '';
+        }
+
+        if (event.target instanceof Element && domEvent.cssLocator) {
+            return domEvent.cssLocator;
+        }
+
+        if (event.target instanceof Node) {
+            return event.target.nodeName;
+        }
+    }
     private addEventHandler(domEvent: TargetDomEvent): void {
         const eventType = domEvent.event;
-        const eventListener = this.getEventListener();
+        const eventListener = this.getEventListener(domEvent);
         this.eventListenerMap.set(domEvent, eventListener);
 
         if (domEvent.elementId) {
             document
                 .getElementById(domEvent.elementId)
                 ?.addEventListener(eventType, eventListener);
+        } else if (domEvent.cssLocator) {
```

Comment:
If there are multiple locators in `domEvent`, which locator should be used? I.e., right now it is `elementID -> cssLocator -> element`. We could also throw an error, but I think it's better to have a sane default here instead.

I think we should add a unit test that defines which locator is used in this case.

Reply:
As discussed, we'll add listeners for all fields that are passed in.